src/App.tsx

```diff
@@ -1,61 +1,127 @@
+import { ChangeEvent, MouseEvent, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+const todos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [initTodos, setTodos] = useState(todos);
+  const [title, setTitle] = useState('');
+  const [emptyTitle, setEmptyTitle] = useState(false);
+  const [userId, setUserId] = useState(0);
+  const [emptyUser, setEmptyUser] = useState(false);
```

Comment:
```suggestion
  const [isValidTitle, setIsValidTitle] = useState(false);
  const [userId, setUserId] = useState(0);
  const [isValidUser, setIsValidUser] = useState(false);
```
consider more meaningful naming.
general convenion in JS to name boolean variables with prefix `is` (sometime in some cases `has`)